app/components/app.js

```diff
@@ -8,23 +11,17 @@ export default React.createClass({
     children: React.PropTypes.object.isRequired
   },
 
-  getInitialState() {
-    return AppStore.getState();
-  },
-
-  componentDidMount() {
-    AppStore.listen(this.onChange);
-  },
-
   onChange() {
     this.setState(this.getInitialState());
   },
 
   render() {
     return (
-      <div className='app'>
-        {this.props.children}
-      </div>
+      <Provider store={store}>
```

Comment:
`react-redux`'s way of allowing all react components to access the store, through their `connect` API